src/main/java/com/develhope/spring/vehicles/controller/VehicleController.java

```diff
@@ -0,0 +1,22 @@
+package com.develhope.spring.vehicles.controller;
+
+import com.develhope.spring.vehicles.service.VehicleServices;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/vehicle")
+public class VehicleController {
+    @Autowired
+    private VehicleServices vehicleServices;
+
+
+    @DeleteMapping("/delete/{id}")
+    public ResponseEntity<Void> deleteVehicle(@PathVariable Long id) {
+        vehicleServices.deleteVehicle(id);
+        return ResponseEntity.noContent().build();
```

Comment:
Perché come ResponseEntity .noContent?

Reply:
Meglio usare un not found in questo caso